app/models/collection-provider.ts

```diff
@@ -7,9 +7,6 @@ import ProviderModel from './provider';
 export default class CollectionProviderModel extends ProviderModel {
     @belongsTo('collection')
     primaryCollection!: DS.PromiseObject<CollectionModel> & CollectionModel;
-
```

Comment:
`collection-provider` endpoint does not have a relationship named `collections`.